Release/include/cpprest/base_uri.h

```diff
@@ -224,6 +224,12 @@ namespace web {
         uri() { m_uri = _XPLATSTR("/");};
 
         /// <summary>
+        /// Creates a URI from the given URI components.
+        /// </summary>
+        /// <param name="components">A URI components object to create the URI instance.</param>
+        _ASYNCRTIMP uri(const details::uri_components &components) : m_components(components) { m_uri = m_components.join(); }
```

Comment:
If the function is defined inline, it shouldn't be marked _ASYNCRTIMP. This can be fixed when merging.